### DISCUSSION TITLE
How to disbale key bind for TAB?

### DISCUSSION THREAD

**Discussion by tonmoy998** [LazyVim/LazyVim#3098] [Q&A] [answered]
Sometimes it's annoying for me to use TAB.Suddenly the cursor moves away.How do I disable this?

**Comment by dpetka2001** [comment, marked as answer]
In LazyVim Tab is used in Luasnip for moving across next/previous jump node as you can see in the [docs](https://www.lazyvim.org/plugins/coding#luasnip). To disable them you could do something like 
```lua
  {
    "L3MON4D3/LuaSnip",
    keys = {
      { "<tab>", false, mode = { "i", "s" } },
      { "<s-tab>", false, mode = { "i", "s" } },
    },
  },
```
But this way you won't be able to jump between snippets nodes back and forth. While disabling `<tab>` and `<s-tab>`, you might also want to add some additional keymaps of your liking, so that you can preserve the behavior of jumping back and forth throughout snippets nodes.

**Reply by tonmoy998** [reply]
Thank you
